[RNP Wrapper Component] Switch (#454)
etn-ccis/blui-react-native-component-library: docs/IconSwitch.md

# Icon Switch

> The `<IconSwitch />` is a switch component which renders a icon in switch's handle. We can use this component with icon rendered and without.

Reviewer: Try making it more succinct and without grammar errors. You also used the word "We" in this paragraph, but everywhere else in the readme, the reader is referred to as "You". Try to be more consistent.

```suggestion
The `<IconSwitch />` is a switch component that can optionally renders an icon in the switch's handle.
```
